src/synthesizer/emission_models/stellar/pacman_model.py

```diff
@@ -113,7 +114,7 @@ def __init__(
         self,
         grid,
         tau_v,
-        dust_curve,
+        dust_curve=PowerLaw(),
```

Comment:
Why force a default here? Surely better to make the user think about it? Especially because a `PowerLaw` also assumes a default slope.

Reply:
I'm open to using a different default (Calzetti?), but I think we should provide one. 99% of users will not care.

Reply:
Fair, I can be convinced to use a default. I was working on a "make them choose" ethos through all the models 😂